newIDE/app/src/UI/Theme/DarkTheme/Mosaic.css

```diff
@@ -23,3 +23,20 @@
 .mosaic-gd-dark-theme.mosaic.mosaic-blueprint-theme .mosaic-preview .mosaic-window-toolbar.draggable:hover {
   background: #3c4698 !important;
 }
+
+/* Separator draggable background */
+.mosaic-gd-dark-theme .mosaic-split.-column .mosaic-split-line {
+  height: 2px;
+  border-bottom: 1px solid #888888 !important;
+  border-top: 1px solid #888888 !important;
+  width: 8%;
```

Comment:
Can we use a fixed width in pixels? I don't think a separator has to be stretched according to the size of the screen.
`width: 100px;` or something like this maybe?

Reply:
Yes why not, but it's not very responsive

Reply:
I see responsiveness less about just "stretching" the things on screen, and more about properly stacking columns or showing panels as sliding windows for example :)
Let me explain: it's more about the organisation of the space. Ideally, if GDevelop was fully responsive, it would retract some panels on small screens, display dialogs as full screen and things that are on two columns would be on one column.
On the contrary, things that are the "widgets" on the screen, must be relatively independant from the size of the screen/window. Imagine that buttons in GitHub would increase their width as you resize the window. Would be strange right? Same here for the separator, it's something that should exist with a fixed size.